packages/common/tests/test-StepFunctions.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const test = require('ava');
+const aws = require('../aws');
+const StepFunctions = require('../StepFunctions');
+const { throttleOnce } = require('../test-utils');
+
+test.serial('getExecutionHistory() retries if a ThrottlingException occurs', async (t) => {
+  const sfnBefore = aws.sfn;
+  try {
```

Comment:
would it make sense to throw these setup blocks into a [beforeEach](https://github.com/avajs/ava/blob/master/docs/recipes/test-setup.md)?

Reply:
They're all slightly different (stubbing different `sfn` functions), but I could probably extract them out into a helper function.  I'll do that.

Reply:
Created a helper function.